spec/redirect_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe GovukPersonalisation::Redirect do
```

Comment:
Nitpick: I think all these tests would be slightly clearer in the form

```ruby
expect(url).to eq("#{base_url}...")
```

Then the final test that nothing changes will just be

```ruby
expect(url).to eq(base_url)
```